### ISSUE TITLE
Waypoint ability breaks extra utilities's angel ring when used

### ISSUE THREAD

**Issue by Builderb0y** [ReikaKalseki/Reika_Mods_Issues#785]
Ring must be dropped/picked up in order to fly again. Also breaks the flugel tiara from botania, and the flight belt from runic dungeons, but does NOT break standard creative mode flight.

I've also noticed that this ability seems to fire a login event 64 times when used, as I see `[Server thread/INFO]: DRAGONAPI DEBUG: Player Builderb0y logging in. Performing config check.` appear in my logs 64 times at the moment I use it.

**Comment by ReikaKalseki** [comment]
Does this still happen, seeing as I redid the ability in v17?

**Comment by Builderb0y** [comment]
I stopped using your mods ages ago. I could care less if it's fixed or not.

**Comment by ray121** [comment]
I know this post is pretty dead, but I have just experienced this bug myself, I installed Dragon API, ChromatiCraft, ReactorCraft, ElectriCraft, RotaryCraft, Expanded Redstone, Geostrata and dye trees.
Now whenever I use an item, such as eating food or using a TiC crossbow the flight stops, but not only can i not take off but I can't even double tap space to stop flying, only landing will stop it. I am launching my MC now to see if I get the same console spam.

Edit: I just realized this is not the same issue, sorry for necroing a post not even related, will move it to the correct issue post.
